src/Microsoft.DocAsCode.App/Config/MetadataJsonItemConfig.cs

```diff
@@ -52,5 +52,8 @@ internal class MetadataJsonItemConfig
 
         [JsonProperty("disableDefaultFilter")]
         public bool DisableDefaultFilter { get; set; }
+
+        [JsonProperty("useMultiLevelToc")]
```

Comment:
Consider use an enum over a bool, in the original issue, there are other proposals to remove repeated prefix from nested namespaces, e.g.

```
Microsoft.DocAsCode
    |- Build
    |- Common
```

We don't need to implement it here, but an enum makes room for future enhancements like this.

Reply:
I've added a third option to the enum, `CompactNested`, which does exactly this.

```yml
- uid: Samples.Foo
  name: Samples.Foo
  items:
  - uid: Samples.Foo.Sub
    name: Sub
    items:
    - uid: Samples.Foo.Sub.SubBar
      name: SubBar
    name.csharp: Samples.Foo.Sub
    name.vb: Samples.Foo.Sub
  - uid: Samples.Foo.Bar
    name: Bar

```